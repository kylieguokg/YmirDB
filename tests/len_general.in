set a
set b 1 a
set c 2 b
set d 3 c
set e 4 d
list entries
len a
len b
len c
len d
len e
pop b
list entries
len a
len b
len c
len d
len e
push a 1 2 3 4 5
append b 2
list entries
len a
len b
len c
len d
len e
pluck a 1
append b a
list entries
len a
len b
len c
len d
len e
bye
