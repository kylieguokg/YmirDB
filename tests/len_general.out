> ok

> ok

> ok

> ok

> ok

> e [4 d]
d [3 c]
c [2 b]
b [1 a]
a []

> 0

> 1

> 2

> 3

> 4

> 1

> e [4 d]
d [3 c]
c [2 b]
b [a]
a []

> 0

> 0

> 1

> 2

> 3

> ok

> ok

> e [4 d]
d [3 c]
c [2 b]
b [a 2]
a [5 4 3 2 1]

> 5

> 6

> 7

> 8

> 9

> 5

> ok

> e [4 d]
d [3 c]
c [2 b]
b [a 2 a]
a [4 3 2 1]

> 4

> 9

> 10

> 11

> 12

> bye
